src/anomalib/models/image/patchcore/torch_model.py

```diff
@@ -123,9 +124,8 @@ def __init__(
         ).eval()
         self.feature_pooler = torch.nn.AvgPool2d(3, 1, 1)
         self.anomaly_map_generator = AnomalyMapGenerator()
-
-        self.register_buffer("memory_bank", torch.empty(0))
         self.memory_bank: torch.Tensor
+        self.register_buffer("memory_bank", torch.Tensor())
```

Comment:
is `torch.Tensor()` really needed? `semgrep` and `bandit` don't like it from security and linter perspective.